.github/workflows/buf.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
           token: ${{ secrets.BUF_TOKEN }}
           format: true
           lint: true
+          # Temp disable breaking checks due to upgrade to v0.11.0
+          breaking: false
```

Comment:
You can revert this and just add the `Buf skip breaking` label (which I added to the PR)
```suggestion
```